src/sentry/static/sentry/app/views/settings/account/accountSecurity/accountSecurityEnroll.tsx

```diff
@@ -415,4 +416,9 @@ class AccountSecurityEnroll extends AsyncView<Props, State> {
   }
 }
 
+const StyledQRCode = styled(QRCode)`
+  background: white;
+  padding: 10px;
```

Comment:
can we use `space(2)` or `space(3)` here?

Reply:
yah, sure thing